index.bs

```diff
@@ -2831,8 +2832,10 @@ parameter to the {{CredentialsContainer/create()}} or {{CredentialsContainer/get
     [=Authenticators=] may implement various [[#enum-transport|transports]] for communicating with [=clients=]. This enumeration
     defines hints as to how clients might communicate with a particular authenticator in order to obtain an assertion for a
     specific credential. Note that these hints represent the [=[WRP]=]'s best belief as to how an authenticator may be reached. A
-    [=[RP]=] may obtain a list of transports hints from some [=attestation statement formats=] or via some out-of-band mechanism;
-    it is outside the scope of this specification to define that mechanism.
+    [=[RP]=] will typically learn of the supported transports for a [=public key credential=] via
+    {{AuthenticatorAttestationResponse/getTransports()}}.
+
+    This enumeration is not referenced by other parts of the IDL because that would preclude other values from being used. But it is important for backwards compatibility that [=client platforms=] and [=[RPS]=] handle unknown values. Therefore it exists here for documentation and as a registry. Where transports are represented elsewhere, they are typed as {{DOMString}}s.
```

Comment:
perhaps this paragraph ought to be a Note: ?

Reply:
suggest:

s/ This enumeration / Note: The {{AuthenticatorTransport}} enumeration /
s/ IDL / Web IDL /
s/ used / used without updating this specification and its implementations /
s/ But it / It /
s/ typed as {{DOMString}}s / typed as {{DOMString}}s, e.g., {{PublicKeyCredentialDescriptor/transports}} /

Reply:
Done. (Tweaked the wording slightly at the very end to read better once bikeshed has transformed things to HTML.)